save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -99,20 +135,43 @@ internal class OrganizationController(
     }
 
     /**
-     * @param organization updateOrganization
+     * @param organizationName name of an organization that should be changed
+     * @param organization draft organization that should be saved as an organization with name [organizationName]
      * @param authentication an [Authentication] representing an authenticated request
      * @return response
      */
     @PostMapping("/{organizationName}/update")
-    @Suppress("UnsafeCallOnNullableType")
-    fun updateOrganization(@RequestBody organization: Organization, authentication: Authentication): Mono<StringResponse> = Mono
-        .just(organization)
-        .filter { organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication, it.name, Role.OWNER) }
-        .map {
-            organizationService.updateOrganization(organization)
+    fun updateOrganization(
+        @PathVariable organizationName: String,
```

Comment:
I don't understand why we need an PathVariable _organizationName_ if we already have an Organization object that contains the name.

Reply:
It might be more user-friendly to use organizationName instead of id
In this case id is not required.
But perhaps I am wrong.
If you find it to be a bad practice, I can remove `organizationName` from path

Reply:
That also might be good if we want to provide an ability to change organization name